LICENSE.md

```diff
@@ -52,39 +52,36 @@ The User accepts the following liability:
  - This Mod may alter fundamental parts of the Game.
  - The User is liable for all damages resulting from the use or misuse of the Mod.
 
-3. Right to use
+3. Right to use and examine
 --------
 
 The User is allowed to install the Mod and play without restriction.  
 The User may not limit access to the Mod in any way, except as provided by the Game.  
 
-4. Right to examine
---------
-
-The User may decompile compiled binaries and examine the resulting source code.  
-The User may examine the source code as it exists in public repositories.
+The User may decompile compiled binaries.  
+The User may examine this Mod's source code.
```

Comment:
I'd rather not combine the two sections. Right to use and right to examine are two different things, and we don't combine terms for any other heading. Let's maintain consistency.